install.ps1

```diff
@@ -239,6 +240,22 @@ if (-not $noChecks.IsPresent) {
     } else {
         Write-Host "`t[+] Username '$extractedUsername' does not contain any spaces." -ForegroundColor Green
     }
+
+# Check Internet connectivity and return boolean value, exit if value is 'false'
```

Comment:
This should be indented. But I think it is better to add a linter for this: https://github.com/mandiant/flare-vm/issues/507